examples/run_examples.py

```diff
@@ -1,9 +1,25 @@
+# interactive=False
+# import subprocess
+# import os
+# import matplotlib
```

Comment:
Could you add some explanation for why the whole file is commented out now?  Perhaps a `""" docstring """` at the top indicating what replaced it?